backend/qemu.pm

```diff
@@ -563,7 +563,9 @@ sub start_qemu {
     {
         push(@params, @vgaoptions);
 
-        gen_params @params, "global", "isa-fdc.driveA=" unless $vars->{QEMU_NO_FDC_SET};
+        unless ($vars->{ARCH} eq 'aarch64' || $vars->{ARCH} eq 'arm' || $vars->{QEMU_NO_FDC_SET}) {
```

Comment:
Why `$vars->{ARCH} eq 'arm'`?
I think we don't use this name.

Reply:
Used that to be consistent compared to #L433

Reply:
You are right.

This is os-autotinst. We don't use this name in os-autoinst-distri*.